metabase/metabase
# Consolidate action utilities

### Description

Primarily moving around action utilities (under `metabase/actions`):

* if a utility is defined inside a component/container and is imported by another component/container, the utility is moved to `metabase/actions/utils` (i.e. shared utils)
* if a utility lives in `metabase/actions/utils`, but is only used in one place, it's moved closer to that code

The PR also includes various minor-ish clean-ups (fixing typos, sorting imports, removing unused/redundant code, etc.)

> **Note**
> Note to reviewers: it'd probably make more sense to review changes commit-by-commit as we're touching different things here and I tried to make each commit atomic

### How to verify

1. CI should be green
2. Play around with actions and ensure everything works and looks as expected


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/29289)
